### PR TITLE
[MAN-698] Merge release-1 branch

### PR DESCRIPTION
**Pull request by DavidZemon**
This is in preparation for the compiler upgrade for s5t.

### CODE REVIEW

File: docker-native [hunk 1]
```diff
@@ -22,4 +22,4 @@ docker run -it --rm \
     -v "$HOME/.conan/.conan.db:/home/captain/.conan/.conan.db" \
     -v "${dir}:${dir}" \
     -w "${dir}" \
-    wsbu/toolchain-native:v0.3.1 "$@"
+    wsbu/toolchain-native:v0.3.3 "$@"
```
Comment (DavidZemon) [inline docker-native:25]: Forgot to upgrade the script last time, which is why the version is getting double-bumped

**Review by mattcaron-redlion — DISMISSED** [review]

**Review by mattcaron-redlion — COMMENTED** [review]
Why are we removing gperf?

**Comment by DavidZemon** [comment]
> Why are we removing gperf?

It was causing something to fail to compile. I can't remember what anymore, but I found lots of other people confirming this error on gentoo forums (and elsewhere) and the only solution I found was to remove gperf, compile your thing, then re-install gperf.

My intention here was to remove gperf, see what breaks, and then determine if that error is any easier to work around than the original. So far... nothing has broken. Do you know why it was added originally?

**Comment by mattcaron-redlion** [comment]
> > Why are we removing gperf?
> 
> It was causing something to fail to compile. I can't remember what anymore, but I found lots of other people confirming this error on gentoo forums (and elsewhere) and the only solution I found was to remove gperf, compile your thing, then re-install gperf.
> 
> My intention here was to remove gperf, see what breaks, and then determine if that error is any easier to work around than the original. So far... nothing has broken. Do you know why it was added originally?

No idea. But, you've answered the question.

**Review by mattcaron-redlion — APPROVED** [review]

**Review by pwalkr — COMMENTED** [review]
Why? I remember verbal mention of gperf causing problems *and was explicitly mentioned above, but what uses docbook-utils?

**Comment by DavidZemon** [comment]
> I remember verbal mention of gperf causing problems *and was explicitly mentioned above, but what uses docbook-utils?

I was trying to get module-init-tools to build without having to make changes to base (it was failing while trying to build documentation). I never ended up getting it to work and ended up invoking specific targets to build the package rather than just `make all`, so `docbook-utils` could be removed.

**Review by pwalkr — APPROVED** [review]

**Pull request merged**